### ISSUE TITLE
Is this up to date and does it work on localhost?

### ISSUE THREAD

**Issue by owenfi** [emmettnicholas/MeteorTheGathering#2]
Total Meteor.js newbie, but I ran this project and nothing loads for me.

(First I did `meteor update` and `meteor add standard-app-packages` and then ran on port 3000.)

Checking in console I get:

    W20150126-15:26:09.630(-8) (blaze.js:71) Warning: Assigning helper with `Template.game.show = ...` is deprecated.  Use `Template.game.helpers(...)` instead.
    logging.js:100 W20150126-15:26:09.636(-8) (blaze.js:71) Warning: Assigning helper with `Template.editor.show = ...` is deprecated.  Use `Template.editor.helpers(...)` instead.
    logging.js:100 W20150126-15:26:09.638(-8) (blaze.js:71) Warning: Assigning helper with `Template.searcher.show = ...` is deprecated.  Use `Template.searcher.helpers(...)` instead.
    current.js:42 Uncaught ReferenceError: createNewGame is not defined

Looks like createNewGame() is defined in models.js, looking into how to get that included now.

(ie. is it file level includes, or is there a meteor command that handles all of that, etc.)

Perhaps some startup/run instructions should be in the readme if there is something obvious I'm missing.

**Comment by owenfi** [comment]
Maybe a meteor.js introduced a breaking change pertaining to scoping somewhere in the last 2 years.

I've tried defining various functions in models.js but they never seem available in current.js (which I also renamed main.js to make sure it loaded last). I put functions in Meteor.is_client as well, but still no luck.

Moving the function body to client code works, and it gets a bit further, but then I get permission denied errors pertaining to my models (i.e. can't insert a card into a deck, and can't find current game vars)

**Comment by owenfi** [comment]
Updated the title because it's also not working upon `deploy` either so I don't believe it's just my environment anymore.

I worked around the function not defined error by putting an equivalent function in a method (so it is called asynchronously with a callback), which helped somewhat, but I ran into a bunch of permission denied's, see here: https://github.com/owenfi/MeteorTheGathering/commit/5cd84156c0037905f6dc74dfe6f0420ff6193a44

Is that the expected way to go about fixing it?

Looks like it is probably incompatibility with current frameworks, so I'm curious what the expected framework versions are.

**Comment by emmettnicholas** [comment]
@owenfi This app worked against an old version of Meteor (few years ago), so I wouldn't be if it isn't supported by the latest version. Miraculously it does still seem to be in working form here though: http://thegathering.meteor.com/

I'll test your PR some time when I'm free. Thanks! :)

**Comment by owenfi** [comment]
Yeah, that PR doesn't get it working again so if you go down a different path or don't have time that's entirely fine.

If you have a way of sussing out what version of Meteor it's running on (and what the dependencies are) that might be handy and I might be able to take another stab at porting it to a more recent version.

> Miraculously it does still seem to be in working form here though: http://thegathering.meteor.com/

That's actually how I found it. Testament to the framework that it feels so modern after three years! Shows a lot of promise and I was hoping to make a couple updates like flipping the top half of the canvas so cards on the field face the owner, higher res cards, deck importing, etc.

Totally understand if you're not maintaining this, but then perhaps a slight disclaimer that it doesn't run on latest meteor would keep people from getting their hopes up :)
